docs/guides/alert-manager-configuration.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "Configuring Notification using Cortex AlertManager"
+linkTitle: "AlertManager configuration"
+weight: 10
+slug: AlertManager-configuration
+---
+
+### Context
+
+Cortex AlertManager notification setup follow mostly the syntax of Prometheus AlertManager since it is based on the same codebase.  The following is a description on how to load the configuration setup so that AlertManager can use for notification when an alert event happened.
+
+#### Cortex AlertManager configuration
+
+Cortex AlertManager can be uploaded via [Cortex Set Alertmanager API](https://cortexmetrics.io/docs/api/#set-alertmanager-configuration) or using Grafana Labs [Cortex Tools](https://github.com/grafana/cortex-tools).
+
+Follow the instruction at the `cortextool` link above to download or update to the latest version of the tool.
+
+To obtain the full help of how to use `cortextool` for all commands and flags, use
+`cortextool --help-long`.
+
+The following example shows the steps to upload the configuration to Cortex `alertmanager` using `cortextool`. 
+
+1. Create `.yaml` file to specify the configuration for AlertManager notification characteristic.  
+
+The following is `amconfig.yml`, an example of a configuration for Cortex `alertmanager` to send notification to a Slack channel:
+
+```
+global:
```

Comment:
I'm wondering if we should use a simpler alertmanager config as an example. The purpose of this guide is not to show how Alertmanager works, but how to configure it in Cortex. I've the feeling such a long example config may deviate the focus from what we want to show in this guide.

For example, the following in a SMTP-based receiver:

```
global:
  # The smarthost and SMTP sender used for mail notifications.
  smtp_smarthost: 'localhost:25'
  smtp_from: 'alertmanager@example.org'
  smtp_auth_username: 'alertmanager'
  smtp_auth_password: 'password'

route:
  # A default receiver.
  receiver: send-email

receivers:
  - name: send-email
    email_configs:
      - to: 'someone@localhost'
```

Reply:
Precisely my issue that motivates me to provide a more "real" example.  Lots of use cases are slack based and I spent countless hours trying to find a more "real" life setup to help with my application of Cortex.  So by providing a little more 'real" example,  I hope to help the users sometimes to comb through many many messages and issued to find the answer

Reply:
I understand this, but to me it looks more like a topic for improving the Prometheus alertmanager doc.

Reply:
That's fine by me.
I got my answer :-)